types/hand-crafted/modules/RTC/JitsiTrack.d.ts

```diff
@@ -3,6 +3,15 @@ import { MediaType } from '../../service/RTC/MediaType';
 import TraceablePeerConnection from './TraceablePeerConnection';
 
 export default class JitsiTrack {
+  conference: JitsiConference;
+  stream: MediaStream;
+  track: MediaStreamTrack;
+  type: MediaType;
+  videoType: unknown;
```

Comment:
we have an enum for this value, maybe we can use it here: `VideoType`

Reply:
Done.